ensime_shared/ensime.py

```diff
@@ -141,7 +141,10 @@ def fetch_runtime_paths():
 
         self.debug_thread_id = None
         self.running = True
-        Thread(target=self.queue_poll, args=()).start()
+
+        thread = Thread(target=self.receive_messages, args=())
```

Comment:
Yes, thank you. Bad partial commit on my part. I wish the unit tests weren't broken. :(